vars/launchBuild.groovy

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/groovy
+
+def call(ctx, jobName, jobId, buildName, fromImage, toTag, dockerfile) {
+  def buildSecret = ctx.dockerCfgSecret(ctx, "builder")
+  ctx.openshift.withCluster() {
+    def output = ctx.openshift.raw("new-build",
```

Comment:
We need `openshift.raw()` here since the `openshift.newBuild()` will clobber our `-o json`

Reply:
don't add `-o json` since newBuild will do it for you. `openshift.raw` won't return nice json objects for you like `newBuild` does

Reply:
How? When I was running `newBuild()` it was using `-o name`